ingeniamotion/motion.py

```diff
@@ -697,6 +697,7 @@ def internal_generator_saw_tooth_move(
         direction: int,
         cycles: int,
         frequency: float,
+        gain: float = 1.0,
```

Comment:
Add it in the docstring. In addition, should it be always positive? because if you write a negative value direction argument loses its sense.

Reply:
Done!